sys-apps/xdg-desktop-portal/xdg-desktop-portal-1.18.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson systemd
+
+DESCRIPTION="Desktop integration portal"
+HOMEPAGE="https://flatpak.org/ https://github.com/flatpak/xdg-desktop-portal"
+SRC_URI="https://github.com/flatpak/${PN}/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="geolocation man systemd test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/glib-2.66:2
+	dev-libs/json-glib
+	>=sys-fs/fuse-3.10.0:3[suid]
+	x11-libs/gdk-pixbuf
+	geolocation? ( >=app-misc/geoclue-2.5.3:2.0 )
+	>=media-video/pipewire-0.3:=
+	systemd? ( sys-apps/systemd )
+	test? (
+		dev-python/pytest
```

Comment:
```suggestion
        ${PYTHON_DEPS}
		dev-python/pytest[${PYTHON_USEDEP}]
```

Reply:
> used python-r1 instead because of pkgcheck scan
> 
> ```
> InvalidDepend: version 1.18.0: failed parsing DEPEND: '>=dev-libs/glib-2.66:2 dev-libs/json-glib >=sys-fs
> ```

Sorry that is my bad, for `python-any-r1` (which we should be using for a purely build-time python dependency) we need instead:
```
BDEPEND="
  test? (
    ${PYTHON_DEPS}
    $(python_gen_any_dep '
        dev-python/pytest[${PYTHON_USEDEP}]
     ')
  )
"

... 

python_check_deps() {
        python_has_version "dev-python/pytest[${PYTHON_USEDEP}]"
}
``` 

(And we won't need the `REQUIRED_USE` line)

Sorry I got the eclasses confused.